lib/Pod/To/HTML.pm

```diff
@@ -157,17 +158,48 @@ sub assemble-list-items(:@content, :$node, *% ) {
 
 
 #| Converts a Pod tree to a HTML document.
-sub pod2html($pod, :&url = -> $url { $url }, :$head = '', :$header = '', :$footer = '', :$default-title,
-  :$css-url = '//design.perl6.org/perl.css', :$lang = 'en',
+sub pod2html(
+    $pod,
+    :&url = -> $url { $url },
+    :$head = '',
+    :$header = '',
+    :$footer = '',
+    :$default-title,
+    :$css-url = '//design.perl6.org/perl.css',
+    :$templates,
```

Comment:
There should probably be a sensible default for this; in general, interface changes should be minimized, or we'll deprecate all of a sudden everything that used this.